plugins/flutter_aepcore/lib/src/aepmobile_init_options.dart

```diff
@@ -0,0 +1,29 @@
+class InitOptions {
+  String? appId;
+  String? filePath;
+  bool lifecycleAutomaticTracking;
```

Comment:
Make this optional and avoid setting a default value. This allows the wrapper to defer to the default value provided by the underlying platform.